app/import_data_v2.py

```diff
@@ -55,14 +55,27 @@ def insert_meta(input, type, company_id):
         # Iterating through the agencies in the json file
         for agency in data['agencies']:
 
+         # Validate region
+            if agency['region'].title() in ['Remote', 'Drenthe', 'Flevoland', 'Friesland', 'Gelderland', 'Groningen', 'Limburg', 'Noord-Brabant', 'Noord-Holland', 'Overijssel', 'Utrecht', 'Zuid-Holland', 'Zeeland']:
+                region = agency['region'].title()
+            else:
+                region = 'Remote'
+
+            # Validate company_size
+            if agency['companySize'] in ['1-10', '11-50', '51-100', 'GT-100']:
+                company_size = agency['companySize']
+            else:
+                company_size = '1-10'  # as default value, or do we want something like 'Unknown'?
```

Comment:
Unknown will be hard to search through .. 

Same here as region -> just reject any records that don't have this